interfaces/builtin/x11_test.go

```diff
@@ -113,28 +114,6 @@ func (s *X11InterfaceSuite) TestAppArmorSpec(c *C) {
 	c.Assert(spec.SnippetForTag("snap.x11.app1"), testutil.Contains, "capability sys_tty_config,")
 }
 
-func (s *X11InterfaceSuite) TestAppArmorSpecOnClassic(c *C) {
-	// on a classic system with x11 slot coming from the core snap.
-	restore := release.MockOnClassic(true)
-	defer restore()
-
-	// connected plug to classic slot
-	spec := &apparmor.Specification{}
-	c.Assert(spec.AddConnectedPlug(s.iface, s.plug, s.classicSlot), IsNil)
-	c.Assert(spec.SecurityTags(), DeepEquals, []string{"snap.consumer.app"})
-	c.Assert(spec.SnippetForTag("snap.consumer.app"), testutil.Contains, "owner /run/user/[0-9]*/.Xauthority r,")
-
-	// connected classic slot to plug
-	spec = &apparmor.Specification{}
-	c.Assert(spec.AddConnectedSlot(s.iface, s.plug, s.classicSlot), IsNil)
-	c.Assert(spec.SecurityTags(), HasLen, 0)
-
-	// permanent classic slot
-	spec = &apparmor.Specification{}
-	c.Assert(spec.AddPermanentSlot(s.iface, s.classicSlotInfo), IsNil)
-	c.Assert(spec.SecurityTags(), HasLen, 0)
-}
-
```

Comment:
Please adjust to add new tests. You can see how that was done in avahi_observe_test.go.

Reply:
today, at the moment. i didn't understand. maybe tomorrow ;-)

Reply:
ok. today, i think i have fixed. Not sure.